generators/client/templates/angular/src/main/webapp/app/blocks/interceptor/_auth-expired.interceptor.ts

```diff
@@ -36,81 +36,86 @@ import { LoginModalService } from '../../shared/login/login-modal.service';
 import { StateStorageService } from '../../shared/auth/state-storage.service';
 <%_ } _%>
 
-export class AuthExpiredInterceptor extends JhiHttpInterceptor {
+export class AuthExpiredInterceptor implements HttpInterceptor {
 
-<%_ if (authenticationType === 'jwt' || authenticationType === 'uaa') { _%>
+<%_ if (authenticationType === 'jwt') { _%>
+    constructor(private injector: Injector) {
+    }
+<%_ } else if (authenticationType === 'uaa') { _%>
     constructor(private injector: Injector) {
-        super();
     }
 <%_ } else if (authenticationType === 'session') { _%>
     constructor(
         private injector: Injector,
-        private stateStorageService: StateStorageService,
-        private loginServiceModal: LoginModalService) {
```

Comment:
what happened to `loginServiceModal`, it is used below I guess you missed it

Reply:
nope its retrieved via injector to fix cyclic dependecy in HTTP_INTERCEPTORS like it was done in line 76 in this file for oauth2 auth type. Anyway i can merge the constructors